### PR TITLE
updating akka and kafka versions

### PR DESCRIPTION
**Pull request by andrea-putzu**
Avoiding "Could not initialize class reactor.netty.http.client.HttpClientSecure" error

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -7,6 +7,8 @@ ThisBuild / organizationName    := "Pagopa S.p.A."
 ThisBuild / libraryDependencies := Dependencies.Jars.`server`
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
+val AkkaVersion = "2.6.18"
```
Comment (galales) [inline build.sbt]:
```suggestion
```
this is already defined in `project/Versions.scala`

File: build.sbt [hunk 2]
```diff
@@ -108,7 +110,8 @@ lazy val kafkaManager = project
     scalafmtOnCompile   := true,
     libraryDependencies := Dependencies.Jars.client ++
       Seq(
-        "com.typesafe.akka" %% "akka-stream-kafka" % "2.1.0" % Compile,
+        "com.typesafe.akka" %% "akka-stream-kafka" % KafkaVersion % Compile,
+        "com.typesafe.akka" %% "akka-stream" % AkkaVersion
```
Comment (galales) [inline build.sbt]:
```suggestion
        "com.typesafe.akka" %% "akka-stream" % akkaVersion
```

File: build.sbt [hunk 3]
```diff
@@ -7,6 +7,8 @@ ThisBuild / organizationName    := "Pagopa S.p.A."
 ThisBuild / libraryDependencies := Dependencies.Jars.`server`
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
+val AkkaVersion = "2.6.18"
+val KafkaVersion = "3.0.0"
```
Comment (galales) [inline build.sbt]: for consistency I suggest to move this value to `project/Versions.scala`

**Review by galales — CHANGES_REQUESTED** [review]

**Comment by andrea-putzu** [comment]
does the fix is ok?

File: project/Versions.scala [hunk 4]
```diff
@@ -17,6 +17,7 @@ object Versions {
   lazy val scalatestVersion         = "3.2.12"
   lazy val slickProjectionVersion   = "1.2.2"
   lazy val nettyVersion             = "2.0.54.Final"
+  lazy val kafkaVersion             = "2.1.1"
```
Comment (TonioGela) [inline project/Versions.scala]:
```suggestion
  lazy val akkaStreamKafkaVersion             = "2.1.1"
```

File: build.sbt [hunk 5]
```diff
@@ -108,7 +108,8 @@ lazy val kafkaManager = project
     scalafmtOnCompile   := true,
     libraryDependencies := Dependencies.Jars.client ++
       Seq(
-        "com.typesafe.akka" %% "akka-stream-kafka" % "2.1.0" % Compile,
+        "com.typesafe.akka" %% "akka-stream-kafka" % kafkaVersion % Compile,
```
Comment (TonioGela) [inline build.sbt]:
```suggestion
        "com.typesafe.akka" %% "akka-stream-kafka" % akkaStreamKafkaVersion % Compile,
```

File: build.sbt [hunk 6]
```diff
@@ -1,4 +1,5 @@
 import ProjectSettings.ProjectFrom
+import Versions.{akkaVersion, kafkaVersion}
```
Comment (TonioGela) [inline build.sbt]:
```suggestion
import Versions.{akkaVersion, akkaStreamKafkaVersion}
```

**Review by TonioGela — CHANGES_REQUESTED** [review]

**Pull request merged**